README.md

```diff
@@ -11,21 +11,19 @@ Dev setup
 Voraussetzungen:
 
 - Python 3
-- Pip
+- pip
+- pipenv
 - PostgreSQL
 
 Datenbank:
 
     createdb interna
 
-[Virtualenv](https://docs.python.org/3/library/venv.html) erstellen:
-
-    python3 -m venv VENV
-    source VENV/bin/activate
-
+[Virtualenv](https://docs.python.org/3/library/venv.html) erstellen und
 Abhängigkeiten installieren:
 
-    pip install -r requirements.txt
+    pipenv install --dev
+    pipenv shell
```

Comment:
Wie bekommt man eigentlich pipenv installiert? pip install --user pipenv? Oder sudo pip install pipenv?

Aber die Anleitung können wir auch in einem separaten PR mal fixen.

Reply:
Ich hab's via pacman installiert :) Ansonsten wäre `pip install --user` wohl keine schlechte Idee.